core/src/main/java/io/questdb/cairo/CairoEngine.java

```diff
@@ -63,6 +66,10 @@ public CairoEngine(CairoConfiguration configuration, @Nullable MessageBus messag
         this.writerPool = new WriterPool(configuration, messageBus);
         this.readerPool = new ReaderPool(configuration);
         this.writerMaintenanceJob = new WriterMaintenanceJob(configuration);
+        this.telemetryQueue = new RingQueue<>(TelemetryRow::new, configuration.getTelemetryQueueCapacity());
+        this.telemetryPubSeq = new SPSequence(configuration.getTelemetryQueueCapacity());
+        this.telemetrySubSeq = new MCSequence(configuration.getTelemetryQueueCapacity());
```

Comment:
this is perhaps incorrect. This queue should be Multiple publisher single consumer